### PR TITLE
Add permissions and user management

### PR DESCRIPTION
**Pull request by ivanaur**
Opened pull request for @ssamardzic's draft.

### CODE REVIEW

**Comment by shiroyasha** [comment]
Replacing @darkofabijan and myself with @milica-nerlovic and @radwo. They have deep knowledge about the topic.

File: user-management-and-permissions_5c07ccc604286304a71d0252.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## Owners
+On Semaphore 2.0, only the creator of the organization can be its Owner. There cannot be multiple owners, but it is possible to transfer the ownership if it's needed. In order to do that, please contact us at [support@semaphoreci.com](mailto:support@semaphoreci.com).
+
+Projects:
+
+The Owner can:
+- See all projects
+- Add projects to Semaphore 2.0, if they have admin access for a particular repository on GitHub
+- Delete their own projects
+- Delete other members’ projects, if they have admin access for a particular repository on GitHub
```
Comment (milica-nerlovic) [inline user-management-and-permissions_5c07ccc604286304a71d0252.md]: I believe that currently the case is that any member can delete any project.

File: user-management-and-permissions_5c07ccc604286304a71d0252.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+Semaphore 2.0 is "mirroring" your GitHub organization and permissions access to the projects is in sync with your GitHub permissions.
```
Comment (milica-nerlovic) [inline user-management-and-permissions_5c07ccc604286304a71d0252.md]: This is not entirely true atm. We do mimick permissions in terms of "does the user have access to this project". But we don't mimick permissions in terms of "what type of access does the user have on this project". Atm, all members have the same access to projects on Semaphore, regardless the level of access on Github. But, @radwo can double check this. :slightly_smiling_face:

Comment (radwo) [reply user-management-and-permissions_5c07ccc604286304a71d0252.md]: Milica is right :+1: if you can see (read access) the project on GH, you have access to it on Semaphore.

**Comment by shiroyasha** [comment]
@ivanaur I have one tiny feedback on this document. 

We should strive reduce the number of times that the word `Semaphore 2.0` appears in the documents. It is obvious by reading the docs that everything here is related to Semaphore 2.0.

**Pull request merged**